proxy/http3/Http3App.h

```diff
@@ -106,6 +106,6 @@ class Http3SettingsFramer : public Http3FrameGenerator
 
 private:
   NetVConnectionContext_t _context;
-  bool _is_done = false; ///< Becarefull when set FIN flag on CONTROL stream. Maybe never?
+  bool _is_done = false; ///< be careful when set FIN flag on CONTROL stream. Maybe never?
```

Comment:
Probably good to capitalize that first 'b' here to be consistent with the next comment.